src/Symfony/Component/Form/Extension/Core/DataTransformer/MoneyToLocalizedStringTransformer.php

```diff
@@ -80,7 +87,7 @@ public function reverseTransform($value)
         $value = parent::reverseTransform($value);
 
         if (null !== $value) {
-            $value *= $this->divisor;
+            $value = function_exists('bcmul') ? bcmul($value, $this->divisor, $this->scale) : $value * $this->divisor;
```

Comment:
This changes the return value to string (which bcmath works with) which could be a bc break.